handlers/read.go

```diff
@@ -14,10 +14,19 @@ func MakeReadHandler(client *containerd.Client, serviceMap *ServiceMap) func(w h
 
 		res := []types.FunctionStatus{}
 		for _, k := range serviceMap.Keys() {
+			i, _ := serviceMap.containers[k].Image(serviceMap.context)
```

Comment:
serviceMap.context? is that just a context.Context? Why not create one here for use?